src/SprykerEco/Zed/UnzerApi/Dependency/External/UnzerApiToGuzzleHttpClientAdapter.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Dependency\External;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\RequestOptions;
+use Psr\Http\Message\ResponseInterface;
+
+class UnzerApiToGuzzleHttpClientAdapter implements UnzerApiToHttpClientInterface
+{
+    /**
+     * @var int
+     */
+    protected const DEFAULT_TIMEOUT = 40;
+    /**
+     * @var string
+     */
+    protected const HEADER_CONTENT_TYPE_KEY = 'Content-Type';
+    /**
+     * @var string
+     */
+    protected const HEADER_CONTENT_TYPE_VALUE = 'application/json';
+
+    /**
+     * @var \GuzzleHttp\Client
+     */
+    protected $guzzleHttpClient;
+
+    public function __construct()
+    {
+        $this->guzzleHttpClient = new Client([
+            RequestOptions::TIMEOUT => static::DEFAULT_TIMEOUT,
+            RequestOptions::HEADERS => [
+                static::HEADER_CONTENT_TYPE_KEY => static::HEADER_CONTENT_TYPE_VALUE,
+            ],
+        ]);
+    }
+
+    /**
+     * @param string $url
+     * @param string $method
+     * @param string $body
+     * @param string $authKey
+     *
+     * @throws \SprykerEco\Zed\UnzerApi\Dependency\External\UnzerApiToHttpClientException
+     *
+     * @return \SprykerEco\Zed\UnzerApi\Dependency\External\UnzerApiToHttpResponseInterface
+     */
+    public function sendRequest(string $url, string $method, string $body, string $authKey): UnzerApiToHttpResponseInterface
```

Comment:
Maybe its better to use here `UnzerApiRequestInterface` instead of those 4 params. Or I miss something.

Reply:
It is adapter to http client implementation so should use more low-level parameters